vignettes/input_preparation_workflow.Rmd

```diff
@@ -148,8 +149,13 @@ We will filter the data and use `tidyr`'s `pivot_longer()` to change the format
 # To read in your own data you can use read_protti()
 protein_groups  <- read_protti(filename = "yourpath/proteinGroups.txt")
 
-protein_groups_filtered <- mq_prot_selected %>%
-  filter(is.na(only_identified_by_site)) %>%  # remove proteins only identified by PTM, reverse hits and potential contaminants
+protein_groups_filtered <- protein_groups %>%
+  mutate(is_only_identified_by_site = ifelse(only_identified_by_site == "+", TRUE, FALSE)) %>%
+  filter(is_only_identified_by_site == FALSE) %>% # remove proteins only identified by PTM
+  mutate(is_potential_contaminant = ifelse(potential_contaminant == "+", TRUE, FALSE)) %>%
+  filter(is_potential_contaminant == FALSE) %>% # remove potential contaminants
```

Comment:
would either overwrite the `potential_contaminant` column with the logical or at least name it like the column in the other part: `is_contaminant`.

But why exactly do you first change them with mutate into a logical and then filter them out? you could just filter them out directly right?

Reply:
In this case, I thought it would be easier for people who would like to understand what we're doing to do it that way.

Reply:
But this is not the way anyone would ever do it because you can filter it out directly. In the end people think you need to do it like this to be able to filter things out.

Reply:
yeah but this column you can also use directly for the qc_contaminants function

Reply:
yes for contaminants but you also filter them out directly here. Then I would at least keep them in afterwards. The other ones I would filter out directly and not first create a new column.